scripts/utils/InsertNuGetIntoSdkAndCli.ps1

```diff
@@ -1,17 +1,17 @@
 ﻿<#
```

Comment:
Now with three project, consider renaming this file to `InsertNuGetIntoDotnet.ps1`

Reply:
This creates a lot of tail because there are dependencies to this in the release scripts. 

Let me check how much work that is before doing it.

Reply:
It's not too bad. I will change it.

Reply:
the problem with changing it is that the release pipeline needs to change, and that means if we need to service old releases the release will fail unless we rename the script on the old branches.

Not saying we shouldn't do it, but it's something we should take into consideration when renaming.

Reply:
oh duh. 
That's why I left certain things the way they are. 

Good point Andy. 
I reneged on decisions that made last Friday when I worked this originally.

Reply:
I have to straight everything out again :)
The trick is that the releases are bound to the dev branch. So the change would've been fine. 
We should think about removing the dev branch dependency from the release and use the branch used to build the code.

Reply:
Thinking about my previous comment, it is true in the generic sense. However with regards to inserting into the CLI, SDK and Toolset repos (the scenarios that use this `ps1` file), we'll only ever do it from the latest dev, or occasionally latest release branch. We should never need to service old releases and use this insertion script.

So there shouldn't be any issue renaming the script, if desired.